scripts/download_hash.sh

```diff
@@ -159,6 +159,43 @@ function get_containerd_archive_checksums {
     done
 }
 
+function get_crio_archive_checksums {
+    declare -A commit_ids=(
+["1.27.0"]="90a1e14a19422dfca437e2d3a95cf73c5412b232"
+)
+
+    echo "crio_archive:" | tee --append "$checksums_file"
+    for arch in arm64 amd64; do
+        echo "  $arch:" | tee --append "$checksums_file"
+        for version in "${@}"; do
+            if _vercmp "${version#v}" '<' "1.27"; then
+                checksum=$(_get_checksum "crio_archive" "$version" "$arch")
+            else
+                url="https://storage.googleapis.com/cri-o/artifacts/cri-o.$arch.${commit_ids[${version#v}]}.tar.gz"
+                checksum=$(_get_checksum "crio_archive" "$version" "$arch" "linux" "$url")
+            fi
+            echo "    ${version#v}: $checksum" | tee --append "$checksums_file"
+        done
+    done
+}
+
+function get_k8s_checksums {
+    local binary=$1
+
+    echo "${binary}_checksums:" | tee --append "$checksums_file"
+    echo "  arm:" | tee --append "$checksums_file"
+    for version in "${@:2}"; do
+        _vercmp "${version#v}" '<' "1.27" && checksum=$(_get_checksum "$binary" "$version" "$arch") || checksum=0
+        echo "    ${version#v}: $checksum" | tee --append "$checksums_file"
+    done
+    for arch in arm64 amd64 ppc64le; do
+        echo "  $arch:" | tee --append "$checksums_file"
+        for version in "${@:2}"; do
+            echo "    ${version#v}: $(_get_checksum "$binary" "$version" "$arch")" | tee --append "$checksums_file"
```

Comment:
HI @electrocucaracha 
It makes kubespray can not install in offline environments.

Reply:
The purpose of this script is to facilitate the maintenance tasks, that means it's going to be used once in a while to collect checksums and submit PR with the latest values. This is not pretend to be used during execution